src/PeopleDashboard.php

```diff
@@ -87,453 +80,451 @@
-          <div class="card-tools pull-right">
-              <button type="button" class="btn btn-tool" data-card-widget="collapse"><i class="fa fa-minus"></i></button>
-              <button type="button" class="btn btn-tool" data-card-widget="remove"><i class="fa fa-times"></i></button>
-          </div>
-      </div>
-      <div class="card-body">
-        <p> <a class="MediumText" href="members/self-register.php"><?php echo gettext('Self Register') ?> <?= gettext('Reports') ?></a>
-        <br>
-        <?php echo gettext('List families that were created via self registration.') ?>
-       </p>
-        <a class="MediumText" href="GroupReports.php"><?php echo gettext('Reports on groups and roles'); ?></a>
-        <br>
-        <?php echo gettext('Report on group and roles selected (it may be a multi-page PDF).'); ?>
-        </p>
-        <?php if (AuthenticationManager::getCurrentUser()->isCreateDirectoryEnabled()) {
-            ?>
-          <p><a class="MediumText"
-                href="DirectoryReports.php"><?= gettext('People Directory') ?></a><br><?= gettext('Printable directory of all people, grouped by family where assigned') ?>
-          </p>
-            <?php
-        } ?>
-        <a class="MediumText" href="LettersAndLabels.php"><?php echo gettext('Letters and Mailing Labels'); ?></a>
-        <br><?php echo gettext('Generate letters and mailing labels.'); ?>
-        </p>
-      </div>
+    <div class="col-lg-6">
+        <div class="card card-info">
+            <div class="card-header with-border">
+                <h3 class="card-title"><i class="fa fa-file-lines"></i> <?= gettext('Reports') ?></h3>
+                <div class="card-tools pull-right">
+                    <button type="button" class="btn btn-tool" data-card-widget="collapse"><i class="fa fa-minus"></i></button>
+                    <button type="button" class="btn btn-tool" data-card-widget="remove"><i class="fa fa-times"></i></button>
+                </div>
+            </div>
+            <div class="card-body">
+                <p> <a class="MediumText" href="members/self-register.php"><?php echo gettext('Self Register') ?> <?= gettext('Reports') ?></a>
+                    <br>
+                    <?php echo gettext('List families that were created via self registration.') ?>
+                </p>
+                <a class="MediumText" href="GroupReports.php"><?php echo gettext('Reports on groups and roles'); ?></a>
+                <br>
+                <?php echo gettext('Report on group and roles selected (it may be a multi-page PDF).'); ?>
+                </p>
+                <?php if (AuthenticationManager::getCurrentUser()->isCreateDirectoryEnabled()) {
+                ?>
+                    <p><a class="MediumText" href="DirectoryReports.php"><?= gettext('People Directory') ?></a><br><?= gettext('Printable directory of all people, grouped by family where assigned') ?>
```

Comment:
Indenting to match the DOM hierarchy might make this easier to read
```suggestion
                    <p>
                        <a class="MediumText" href="DirectoryReports.php"><?= gettext('People Directory') ?></a>
                        <br>
                        <?= gettext('Printable directory of all people, grouped by family where assigned') ?>
```

Reply:
Not sure what you mean, but those `<br>` need to go. Noticed them yesterday, but was planning that for a separate PR.

Reply:
rephrased: how about splitting the `<p>` and `<a>` tags to make this more readable?